plaidml/bridge/tensorflow/tests/i3d_test.cc

```diff
@@ -25,341 +72,405 @@ string I3DTestSpecToString(const ::testing::TestParamInfo<I3DTestSpec>& info) {
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_0/Conv3d_0a_1x1/batch_norm/beta:0"),             // %arg78
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_0/Conv3d_0a_1x1/batch_norm/moving_mean:0"),      // %arg79
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_0/Conv3d_0a_1x1/batch_norm/moving_variance:0"),  // %arg80
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_0/Conv3d_0a_1x1/conv_3d/w:0"),                   // %arg81
+      std::vector<float>{0.001},  // %arg82: RGB/inception_i3d/Mixed_3b/Branch_1/Conv3d_0a_1x1/batch_norm/batch_norm/add
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_1/Conv3d_0b_3x3/batch_norm/beta:0"),             // %arg83
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_1/Conv3d_0b_3x3/batch_norm/moving_mean:0"),      // %arg84
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_1/Conv3d_0b_3x3/batch_norm/moving_variance:0"),  // %arg85
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_1/Conv3d_0b_3x3/conv_3d/w:0"),                   // %arg86
+      std::vector<float>{0},      // %arg87: RGB/inception_i3d/Mixed_3b/Branch_1/Conv3d_0a_1x1/Relu
+      std::vector<float>{0.001},  // %arg88: RGB/inception_i3d/Mixed_3b/Branch_1/Conv3d_0a_1x1/batch_norm/batch_norm/add
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_1/Conv3d_0a_1x1/batch_norm/beta:0"),             // %arg89
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_1/Conv3d_0a_1x1/batch_norm/moving_mean:0"),      // %arg90
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_1/Conv3d_0a_1x1/batch_norm/moving_variance:0"),  // %arg91
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_1/Conv3d_0a_1x1/conv_3d/w:0"),                   // %arg92
+      std::vector<float>{0.001},  // %arg93: RGB/inception_i3d/Mixed_3b/Branch_2/Conv3d_0a_1x1/batch_norm/batch_norm/add
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_2/Conv3d_0b_3x3/batch_norm/beta:0"),             // %arg94
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_2/Conv3d_0b_3x3/batch_norm/moving_mean:0"),      // %arg95
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_2/Conv3d_0b_3x3/batch_norm/moving_variance:0"),  // %arg96
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_2/Conv3d_0b_3x3/conv_3d/w:0"),                   // %arg97
+      std::vector<float>{0},      // %arg98: RGB/inception_i3d/Mixed_3b/Branch_2/Conv3d_0a_1x1/Relu
+      std::vector<float>{0.001},  // %arg99: RGB/inception_i3d/Mixed_3b/Branch_2/Conv3d_0b_3x3/batch_norm/batch_norm/add
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_2/Conv3d_0a_1x1/batch_norm/beta:0"),             // %arg100
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_2/Conv3d_0a_1x1/batch_norm/moving_mean:0"),      // %arg101
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_2/Conv3d_0a_1x1/batch_norm/moving_variance:0"),  // %arg102
+      lookup("RGB/inception_i3d/Mixed_3b/Branch_2/Conv3d_0a_1x1/conv_3d/w:0"),                   // %arg103
+      std::vector<float>{
+          0.001},  // %arg104: RGB/inception_i3d/Mixed_3c/Branch_0/Conv3d_0a_1x1/batch_norm/batch_norm/add
```

Comment:
This isn't a blocker for merging, but it might be nice to have a cleaner way to format these.